src/PrestaShopBundle/Api/QueryParamsCollection.php

```diff
@@ -608,6 +615,35 @@ protected function appendSqlActiveFilterParam($value, $sqlParams)
         return $sqlParams;
     }
 
+    /**
+     * @param array $filters
+     * @param string|int $only_restockable
+     *
+     * @return array
+     */
+    protected function appendSqlOnlyRestockableFilter(array $filters, $only_restockable)
```

Comment:
Sorry I don't understand, are you intend to use a more efficient function name or use simpler function name and add annotation comment before function declaration?

Reply:
Sorry for the unclear comment ^^'
In fact I brought two subjects : 
1. A suggestion : adding `: array` to function signature
2. A question : if you add `: array` to function signature, i asked it would be necessary to keep @return annotation ?